src/containers/LoginForm/index.jsx

```diff
@@ -0,0 +1,2 @@
+import LoginForm from './LoginForm';
+export default LoginForm;
```

Comment:
Probably a better way to do it...
I wanted to avoid the double `LoginForm/LoginForm` import from other components.
I thought a `export * from './LoginForm'` would have worked, but I'm still missing something about imports with webpack...
@jrmi any idea?